Cargo.toml

```diff
@@ -31,11 +31,6 @@ members = [
 	"runtime/moonriver",
 ]
 
-[patch.crates-io]
-jsonrpsee = { git = "https://github.com/PureStake/jsonrpsee", branch = "moonbeam-v0.14" }
-jsonrpsee-core = { git = "https://github.com/PureStake/jsonrpsee", branch = "moonbeam-v0.14" }
-jsonrpsee-types = { git = "https://github.com/PureStake/jsonrpsee", branch = "moonbeam-v0.14" }
-
```

Comment:
@librelois do we still need a custom branch for this?

Reply:
(note that we are updating from `jsonrpsee` from `0.14` to `0.15`)

Reply:
@notlesh yes, please use the custom branch `moonbeam-v0.15` that I just created, this include logging for HTTP request (already merged upstream but only in jsonrpsee v0.2).